src/runtime/classobj.cpp

```diff
@@ -840,6 +966,78 @@ Box* instanceDelslice(Box* _inst, Box* i, Box* j) {
     }
 }
 
+// Analoguous to CPython's, used for sq_slots.
+static int instance_ass_slice(Box* inst, Py_ssize_t i, Py_ssize_t j, PyObject* value) {
+    static BoxedString* delslice_str, *setslice_str, *delitem_str, *setitem_str;
+    Box* func = NULL, * res = NULL;
+    Box* slice = NULL;
+    Box* begin = boxInt(i);
```

Comment:
`begin` and `end` will get leaked at the `return -1` statments.
better use `AUTO_DECREF(begin);`